src/librustc_metadata/creader.rs

```diff
@@ -302,10 +302,13 @@ impl<'a> CrateLoader<'a> {
             crate_root.def_path_table.decode(&metadata)
         });
 
+        let exported_symbols = crate_root.exported_symbols.decode(&metadata).collect();
```

Comment:
Huh, I thought I had removed this... At least it's by `DefIndex` not by symbol name.